src/Entity/Adherent.php

```diff
@@ -1074,4 +1074,18 @@ public function getOAuthUser(): InMemoryOAuthUser
 
         return $this->oAuthUser;
     }
+
+    public function serialize()
+    {
+        return serialize([
+            $this->id,
+            $this->emailAddress,
+            $this->password,
+        ]);
+    }
+
+    public function unserialize($serialized)
+    {
+        list($this->id, $this->emailAddress, $this->password) = unserialize($serialized);
```

Comment:
Why not using short syntax ?
```php
[$this->id, .........] = ....
```

Reply:
@cacahouwete no particular reason, except that `list` is clearer to me